packages/terra-card/src/Card.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './Card.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Provides themeable 10px on top and bottom
```

Comment:
Are we keeping the padding on the Card? https://github.com/cerner/terra-core/pull/721#discussion_r133001319